src/main.rs

```diff
@@ -44,7 +46,7 @@ fn main() {
     info!("Starting server...");
 
     /* Should be port 3000 eventually */
-    let addr = "127.0.0.1:1337".parse().unwrap();
+    let addr = ([127, 0, 0, 1], 3000).into();
```

Comment:
Changing port number is not in scope for this PR and would need the comment above removed.

Reply:
removed changed of port number